Doc/c-api/object.rst

```diff
@@ -196,6 +196,17 @@ Object Protocol
       This function now includes a debug assertion to help ensure that it
       does not silently discard an active exception.
 
+
+.. c:function:: PyObject* _PyObject_FunctionStr(PyObject *func)
+
+   Return a user-friendly string representation of the function-like object
+   *func*. This returns ``func.__qualname__ + "()"`` if there is a
+   ``__qualname__`` attribute and ``type(func).__name__ + " object"``
+   otherwise. Note that there is no check that *func* is actually callable.
```

Comment:
Why not using the qualified name for type(func)?

Would it be possible to also add the module, except for builtin functions?

Reply:
> Why not using the qualified name for type(func)?

I decided to use `str(func)` as fallback, which looks more useful than `f"{type(func)} object"`.